stable/ipfs/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ if .Values.service.nameOverride }} {{ .Values.service.nameOverride }} {{ else }} {{ template "fullname" . }} {{ end }}
```

Comment:
Because I want to be allow users to override service names. In this case, end users will be asked to hit the DNS entry for the service name, and that should be overrideable so it can be something like 'ipfs' so other users in the same namespace can just specify 'ipfs'. The default is still the behavior seen in the current charts in the repo.